app/src/main/java/ch/epfl/sdp/drone3d/ui/auth/LoginActivity.kt

```diff
@@ -42,8 +44,18 @@ class LoginActivity : AuthActivity() {
     }
 
     fun login(@Suppress("UNUSED_PARAMETER") view: View) {
```

Comment:
I just noticed it now but a small comment could be great as it is a public method.